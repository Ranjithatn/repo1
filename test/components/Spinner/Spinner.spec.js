import React from "react";
import toJson from "enzyme-to-json";
import Spinner from "../../../app/components/Spinner/Spinner";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow, mount } from "enzyme";


configure({ adapter: new Adapter() });

describe('<Spinner />', () => {

  it("should render without crashing", () => {
    const component = mount(
      <Spinner  />
    );
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });

});
