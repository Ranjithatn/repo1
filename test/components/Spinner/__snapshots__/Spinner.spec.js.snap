// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> should render without crashing 1`] = `
<Spinner>
  <div
    id="Spinner"
    style={
      Object {
        "zIndex": 50,
      }
    }
  >
    <div
      className="sk-fading-circle"
    >
      <div
        className="sk-circle1 sk-circle"
      />
      <div
        className="sk-circle2 sk-circle"
      />
      <div
        className="sk-circle3 sk-circle"
      />
      <div
        className="sk-circle4 sk-circle"
      />
      <div
        className="sk-circle5 sk-circle"
      />
      <div
        className="sk-circle6 sk-circle"
      />
      <div
        className="sk-circle7 sk-circle"
      />
      <div
        className="sk-circle8 sk-circle"
      />
      <div
        className="sk-circle9 sk-circle"
      />
      <div
        className="sk-circle10 sk-circle"
      />
      <div
        className="sk-circle11 sk-circle"
      />
      <div
        className="sk-circle12 sk-circle"
      />
    </div>
  </div>
</Spinner>
`;
